egs/wsj/s5/steps/nnet3/train_discriminative.sh

```diff
@@ -307,24 +303,25 @@ while [ $x -lt $num_iters ]; do
       nnet3-am-copy --set-raw-nnet=- $dir/$x.mdl $dir/$[$x+1].mdl || exit 1;
 
     rm $nnets_list
+    [ ! -f $dir/$[$x+1].mdl ] && echo "$0: Did not create $dir/$[$x+1].mdl" && exit 1;
+    if [ -f $dir/$[$x-1].mdl ] && $cleanup && \
+       [ $[($x-1)%$keep_model_iters] -ne 0  ] && \
```

Comment:
... I'd be OK to have a 'keep_model_iters' flag, default, say, 100, that would make it save more models [but permanently, not temporarily].